src/cljs/coffee_re_frame/pages/recipe_brew.cljs

```diff
@@ -15,6 +15,11 @@
     [c/home-button]]
    [recipe-title recipe]])
 
+
+(defn time-alert [seconds]
```

Comment:
Hey we should instead make this an effect that gets returned from an event handler instead of putting it into the view. Views shouldn't really do anything except update the doom.

Reply:
You could either look up how effect handlers work or I can show you how that works if you'd like